### DISCUSSION TITLE
fontconfig not installing

### DISCUSSION THREAD

**Discussion by catpreneur** [Homebrew/discussions#2302] [Linux] [answered]
fontconfig was failing to install, so i looked at the build log and it complains that `docbook2pdf` command not found, so I googled, and sure enough I found these issues https://github.com/Homebrew/linuxbrew-core/issues/1177 https://github.com/Homebrew/linuxbrew-core/issues/4950
in the thread, people say to run
```
brew install fontconfig --without-docs
```
I tried that, but it doesn't work! still fails! so instead, I edited the fontconfig formula

```
brew edit fontconfig
```

and in the `./configure` line and added `--disable-docs` parameter, and it worked!1! can we update this somehow? I couldn't post a comment in those threads.

**Comment by SMillerDev** [comment]
You can make a pull request to homebrew-core to fix it.

**Comment by catpreneur** [comment]
@SMillerDev Here's my changes https://github.com/Homebrew/linuxbrew-core/compare/master...oofdog:patch-1 but can't open a pull request.

**Reply by SMillerDev** [reply]
That's linuxbrew-core. Please make a pull request against https://github.com/Homebrew/homebrew-core

**Reply by catpreneur** [reply]
Done, https://github.com/Homebrew/homebrew-core/pull/88364

**Comment by iMichka** [comment, marked as answer]
Fixed by https://github.com/Homebrew/homebrew-core/pull/88364
